crates/interpreter/src/host.rs

```diff
@@ -48,8 +47,8 @@ pub trait Host {
     /// Set the transient storage value of `address` at `index`.
     fn tstore(&mut self, address: Address, index: U256, value: U256);
 
-    /// Emit a log owned by `address` with given `topics` and `data`.
-    fn log(&mut self, address: Address, topics: Vec<B256>, data: Bytes);
+    /// Emit a log owned by `address` with given `LogData`.
+    fn log(&mut self, address: Address, log_data: LogData);
```

Comment:
```suggestion
    fn log(&mut self, log: Log);
```

`Log` contains both `Address` and  `LogData`.

@slordua i meant this

Reply:
ah ok i see it thx, will revise shortly